gcd/src/main/java/org/spine3/server/storage/datastore/DsProjectionStorage.java

```diff
@@ -65,7 +66,7 @@ public void writeLastHandledEventTime(Timestamp timestamp) {
     @Override
     public Timestamp readLastHandledEventTime() {
         final Timestamp readTimestamp = propertyStorage.read(lastTimestampId);
-        if (Timestamp.getDefaultInstance().equals(readTimestamp)) {
+        if ((readTimestamp == null) || Validate.isDefault(readTimestamp)) {
```

Comment:
`isDefault()` method also checks for null. Though this is present in docs, I think we'll rename this method to `isDefaultOrNull`, maybe.